handler.go

```diff
@@ -132,153 +130,137 @@ func (ca *Cagent) PostResultsToHub(result Result) error {
 	return nil
 }
 
-func (ca *Cagent) Run(outputFile *os.File, interrupt chan struct{}, once bool) {
+func (ca *Cagent) GetAllMeasurements() (MeasurementsMap, error) {
+	var errs []string
+	var measurements = make(MeasurementsMap)
 
-	var jsonEncoder *json.Encoder
-	if ca.PidFile != "" && !once && runtime.GOOS != "windows" {
-		err := ioutil.WriteFile(ca.PidFile, []byte(strconv.Itoa(os.Getpid())), 0664)
-
-		if err != nil {
-			log.Errorf("Failed to write pid file at: %s", ca.PidFile)
-		}
+	cpum, err := ca.CPUWatcher().Results()
+	if err != nil {
+		// no need to log because already done inside cpu.Results()
+		errs = append(errs, err.Error())
 	}
+	measurements = measurements.AddWithPrefix("cpu.", cpum)
```

Comment:
please either have a new line before `measurements ...` or not

Reply:
fixed